components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3692,5 +3249,628 @@ subroutine get_time_space_phys_variables( &
 
 end subroutine get_time_space_phys_variables
 
+subroutine cloud_sedimentation(kts,kte,    &
+   qc_incld,rho,inv_rho,lcldm,acn,inv_dzq,&
+   dt,odt,dnu,lammin,lammax,log_predictNc, & 
+   qc, nc, nc_incld,mu_c,nu,lamc,prt_liq,qc_tend,nc_tend) 
+   
+   implicit none 
+   integer, intent(in) :: kts, kte
+
+   real(rtype), intent(in), dimension(kts:kte) :: qc_incld
+
+   real(rtype), intent(in), dimension(kts:kte) :: rho
+   real(rtype), intent(in), dimension(kts:kte) :: inv_rho 
+   real(rtype), intent(in), dimension(kts:kte) :: lcldm 
+   real(rtype), intent(in), dimension(kts:kte) :: acn 
+   real(rtype), intent(in), dimension(kts:kte) :: inv_dzq
+   real(rtype), intent(in) :: dt 
+   real(rtype), intent(in) :: odt 
+   real(rtype), dimension(:), intent(in) :: dnu
+   real(rtype), intent(in) :: lammin
+   real(rtype), intent(in) :: lammax 
+   logical, intent(in) :: log_predictNc 
+
+   real(rtype), intent(inout), dimension(kts:kte) :: qc 
+   real(rtype), intent(inout), dimension(kts:kte) :: nc 
+   real(rtype), intent(inout), dimension(kts:kte) :: nc_incld 
+   real(rtype), intent(inout), dimension(kts:kte) :: mu_c 
```

Comment:
is `mu_c` actually changed in sedimentation?  If so, how is it used after sedimentation?

Reply:
same for nu and lamc

Reply:
It looks like making mu_c and  lamc local to the subroutine breaks BFB, while making nu local does not. I have thus made nu a subroutine local variable and have left mu_c and lamc unchanged. We should revisit the correctness of of having cloud_sedimentation modify mu_c and lamc.